Nuget/Ploeh.AutoFixture.nuspec

```diff
@@ -6,6 +6,9 @@
         <title>AutoFixture</title>
         <authors>Mark Seemann</authors>
         <owners>Mark Seemann</owners>
+        <dependencies>
+            <dependency id="Albedo" version="[1,2)" />

```

Comment:
Everything works fine with Albedo 1.0.0.

BTW, there is [AutoFixture.Igioms](https://www.nuget.org/packages/AutoFixture.Idioms) package which is already published with Albedo v1.0.1 dependency. There should be no harm in downgrading this reference to v1.0.0. Please let me know if you have any concerns about this.